component/mcsd/updater.go

```diff
@@ -57,8 +57,13 @@ func buildUpdateTransaction(tx *fhir.Bundle, entry fhir.BundleEntry, allowedReso
 			if err := json.Unmarshal(entry.Resource, &endpoint); err != nil {
 				return "", fmt.Errorf("failed to unmarshal Endpoint resource: %w", err)
 			}
+
 			// Import mCSD directory endpoints even from discoverable directories
-			doSync = coding.EqualsCode(endpoint.ConnectionType, coding.MCSDConnectionTypeSystem, coding.MCSDConnectionTypeDirectoryCode)
+			var payloadCoding = fhir.Coding{
```

Comment:
could you move this as a constant (well it can't be a constant, but declared once, at least) to `coding` package?

Or you could make a defacto constant by making a function for it that returns this